Index: MuinosJoseUnidad72/Ejercicio 3/equipos.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"es\">\r\n\t<head>\r\n\t\t<meta charset=\"UTF-8\">\r\n\t\t<title>Equipos de Fútbol</title>\r\n\t</head>\r\n\t<body>\r\n\t\t<select id=\"equiposSelect\" onchange=\"mostrarDatosEquipo()\">\r\n\t\t</select>\r\n\t\t<div id=\"datosEquipo\">\r\n\t\t</div>\r\n\t\t<script src=\"equipos.js\"></script>\r\n\t</body>\r\n</html>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/MuinosJoseUnidad72/Ejercicio 3/equipos.html b/MuinosJoseUnidad72/Ejercicio 3/equipos.html
--- a/MuinosJoseUnidad72/Ejercicio 3/equipos.html	
+++ b/MuinosJoseUnidad72/Ejercicio 3/equipos.html	
@@ -3,6 +3,8 @@
 	<head>
 		<meta charset="UTF-8">
 		<title>Equipos de Fútbol</title>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+	
 	</head>
 	<body>
 		<select id="equiposSelect" onchange="mostrarDatosEquipo()">
